jwst/clean_flicker_noise/clean_flicker_noise.py

```diff
@@ -969,7 +969,7 @@ def _make_intermediate_model(input_model, intermediate_data):
 
     Returns
     -------
-    intermediate_model : ~jwst.datamodel.JwstDataModel`
+    intermediate_model : `~jwst.datamodel.JwstDataModel`
```

Comment:
This one is also a common mistake that might be useful to find and replace: it should be `datamodels`, not `datamodel`.
```suggestion
    intermediate_model : `~jwst.datamodels.JwstDataModel`
```